builtin/tests/commit_post_test.rs

```diff
@@ -0,0 +1,206 @@
+    let genesis = create_genesis_actors(&mut builder, &spec).unwrap();
+    let bench = builder.build().unwrap();
+    let mut w = ExecutionWrangler::new_default(bench);
+
+    // create an owner account
+    let addrs = create_accounts(
+        &mut w,
+        genesis.faucet_id,
+        1,
+        TokenAmount::from_whole(10_000),
+        SignatureType::BLS,
+    )
+    .unwrap();
+    let (owner, worker) = (addrs[0].clone(), addrs[0].clone());
+    let seal_proof = RegisteredSealProof::StackedDRG32GiBV1P1;
+    let (miner_id, miner_addr) = create_miner(
+        &mut w,
+        owner.id,
+        worker.id,
+        seal_proof.registered_window_post_proof().unwrap(),
+        TokenAmount::from_whole(10_000),
+    )
+    .unwrap();
+    w.set_epoch(200);
+
+    // precommit and advance to prove commit time
+    let sector_number: SectorNumber = 100;
+    precommit_sectors(
+        &mut w,
+        1,
+        1,
+        &worker.id_addr(),
+        &miner_addr,
+        seal_proof,
+        sector_number,
+        true,
+        None,
+    );
+
+    let balances = get_miner_balance(&mut w, miner_id);
+    assert!(balances.pre_commit_deposit.is_positive());
+
+    let prove_time = w.epoch() + Policy::default().pre_commit_challenge_delay + 1;
+    advance_by_deadline_to_epoch(&mut w, &miner_addr, prove_time);
+
+    // prove commit, cron, advance to post time
+    let prove_params = ProveCommitSectorParams { sector_number, proof: vec![] };
+    let _prove_params_ser = IpldBlock::serialize_cbor(&prove_params).unwrap();
+    println!("Running prove_commit_sector");
+    let res = apply_ok(
+        &mut w,
+        worker.id_addr(),
+        miner_addr,
+        TokenAmount::zero(),
+        MinerMethod::ProveCommitSector as u64,
+        &prove_params,
+    )
+    .unwrap();
+    assert_eq!(ExitCode::OK, res.receipt.exit_code, "ProveCommitSector failed {:?}", res);
+
+    println!("Running cron job");
+    let res = w
+        .execute_implicit(
+            SYSTEM_ACTOR_ADDR,
+            CRON_ACTOR_ADDR,
+            CronMethod::EpochTick as u64,
+            RawBytes::default(),
+            TokenAmount::zero(),
+        )
+        .unwrap();
+    assert_eq!(ExitCode::OK, res.receipt.exit_code);
+
+    let analysis = TraceAnalysis::build(res.trace);
+    println!("{}", analysis.format_spans());
+
+    // pcd is released ip is added
+    let balances = get_miner_balance(&mut w, miner_id);
+    assert!(balances.initial_pledge.is_positive());
+    assert!(balances.pre_commit_deposit.is_zero());
+
+    println!("PCD verified, initial pledge is positive {}", balances.initial_pledge);
+
+    // power unproven so network stats are the same
+    // let network_stats = v.get_network_stats();
+    // assert!(network_stats.total_bytes_committed.is_zero());
+    // assert!(network_stats.total_pledge_collateral.is_positive());
+
+    let (deadline_info, partition_index) =
+        advance_to_proving_deadline(&mut w, &miner_addr, sector_number);
```

Comment:
Nit: I'd pull this final advance up into the test itself, since it's not really related to the initial commitment.